### DISCUSSION TITLE
PBPK/PD model of nicotine and cotinine

### DISCUSSION THREAD

**Discussion by LkKov** [Open-Systems-Pharmacology/Forum#731] [Announcement]
Dear OSP-community,

We have recently been working on a PBPK/PD model of nicotine and its major metabolite cotinine. The thoroughly developed parent-metabolite model is able to predict plasma concentrations, nicotine brain tissue concentrations, and heart rate profiles after intravenous, oral, transdermal, and pulmonary nicotine intake. 

The corresponding manuscript, including comprehensive documentation, has been published online in Clinical Pharmacokinetics. The PK-Sim and MoBi model files presenting the PBPK/PD model including simulations of clinical studies can be found [here](https://github.com/Open-Systems-Pharmacology/Nicotine-Cotinine-Model/).

Related publication:
Kovar L, Selzer D, Britz H, Benowitz N, St. Helen G, Kohl Y, Bals R, Lehr T. Comprehensive parent-metabolite PBPK/PD modeling insights into nicotine replacement therapy strategies. Clin Pharmacokinet 2020;Epub:https://doi.org/10.1007/s40262-020-00880-4.

Best regards,
Lukas

**Comment by tobiasK2001** [comment]
Excellent! Thanks for this!
